src/node_process_methods.cc

```diff
@@ -471,6 +473,128 @@ static void ReallyExit(const FunctionCallbackInfo<Value>& args) {
   env->Exit(code);
 }
 
+inline char** copy_js_strings_array(Environment* env,
+                                    Local<Array> js_array,
+                                    int* target_length) {
+  Local<Context> context = env->context();
+  char** target = nullptr;
+  int length = js_array->Length();
+
+  CHECK_LT(length, INT_MAX);
```

Comment:
```suggestion
  CHECK_LT(length, INT_MAX);
  CHECK_BT(length, 0);
```

Reply:
I couldn't find `CHECK_BT` anywhere. Did you mean `CHECK_GT` or what?

Reply:
Yes!

Reply:
The function has been removed, so no need to this change anymore.